internal/boxcli/midcobra/telemetry.go

```diff
@@ -176,6 +181,9 @@ func (m *telemetryMiddleware) trackEvent(evt *event) {
 		_ = segmentClient.Close()
 	}()
 
+	// deliberately ignore error
+	_ = identifyUser(segmentClient, evt.DeviceID)
```

Comment:
Nit: Should we go ahead and rename the `DeviceID` field in the event object to `AnonymousID`?